### PR TITLE
Edit Rust SDK doc, add links to supporting docs

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: doc/src/build/rust-sdk.md [hunk 1]
```diff
@@ -3,25 +3,43 @@ title: Sui Rust SDK
...
 [dependencies]
 sui-sdk = { git = "https://github.com/MystenLabs/sui" }
 ```
-Use the devnet branch if you are connecting to the devnet. 
+If you are connecting to the devnet, use the `devnet` branch instead:
 ```toml
 [dependencies]
 sui-sdk = { git = "https://github.com/MystenLabs/sui", branch = "devnet" }
 ```
+TODO: Give them a little more guidance on what this file is and where to find it. We describe `Move.toml` more fully at:
```
Comment (randall-Mysten) [inline doc/src/build/rust-sdk.md:32]: this is intentionally public viewable?

**Review by randall-Mysten — APPROVED** [review]
Approving with a comment/question about the To Do line

File: doc/src/build/rust-sdk.md (hunk 1 above)
Comment (Clay-Mysten) [reply doc/src/build/rust-sdk.md:32]: > this is intentionally public viewable?

Yep. Let's have Patrick review before merge.

Comment (patrickkuo) [reply doc/src/build/rust-sdk.md:32]: I am expecting the audience of this doc knows the basic of Rust and `Cargo.toml`, I don't think we need more descriptions here?

Comment (Clay-Mysten) [reply doc/src/build/rust-sdk.md]: > I am expecting the audience of this doc knows the basic of Rust and `Cargo.toml`, I don't think we need more descriptions here?

Great. I removed the TODO and linked into the Cargo doc.

**Pull request merged**